multimodal/src/autogluon/multimodal/utils/data.py

```diff
@@ -153,6 +156,12 @@ def create_data_processor(
             model=model,
             max_len=model_config.max_text_len,
         )
+    elif data_type == ROIS:
```

Comment:
In this case, both ocr and detection use `MMDetProcessor `. Need to consider the design.

Reply:
We may also need to add the problem_type here while we work on OCR. The current OCR implementation needs a refactor and unit tests are blocked.